app/src/main/java/com/framgia/wsm/screen/confirmrequestoff/ConfirmRequestOffViewModel.java

```diff
@@ -243,9 +243,53 @@ public String getTitleToolbar() {
         return mContext.getString(R.string.confirm_edit_request_off);
     }
 
+    public String getStartDateHaveSalary() {
+        if (mOffRequest != null && mOffRequest.getStartDayHaveSalary() != null) {
+            if (!mOffRequest.getStartDayHaveSalary().getOffPaidFrom().equals("")) {
```

Comment:
WARNING: [CollapsibleIfStatements] These nested if statements could be combined
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/basic.html#CollapsibleIfStatements